src/java/main/org/apache/zookeeper/server/command/FourLetterCommands.java

```diff
@@ -216,6 +216,10 @@ public static boolean isEnabled(String command) {
                     whiteListedCommands.add(cmd.trim());
                 }
             }
+            // It is sad that isro and srvr are used by ZooKeeper itself. Need fix this
+            // before deprecating 4lw.
+            whiteListedCommands.add("isro");
```

Comment:
I've few comments, please see:

comment-1) I agree that these commands are very much needed for ZK functionality and cannot be disabled. I hope you have coded with that point in mind. In general, default values can be overridden, but here these are mandatory values. Can we document conveying these thoughts. Presently the documentation says that `"The default value is empty, which disables all Four Letter Words command."`

comment-2) Say, user keeps `4lw.commands.whitelist=<empty>`, then `System.getProperty(ZOOKEEPER_4LW_COMMANDS_WHITELIST)` will be null and these two commands will not be added to `whiteListedCommands`, right?